docs/getting-started/certificate-pinning.adoc

```diff
@@ -0,0 +1,47 @@
+== Certificate Pinning
+
+Certificate pinning can be enabled in individual SDKs through the `mobile-services.json` file.
+Certificate pinning will only be enabled for services which are used directly by the SDKs. For
+internal services SPKI pinning is used. External services must have pinning configured
```

Comment:
not sure if it is clear to developers what "internal" and "external" services mean. I think it might be better to use "mobile services" and "other services"?